### DISCUSSION TITLE
Multiple Autounattend.xml files on Ventoy

### DISCUSSION THREAD

**Discussion by BYTEMEE1** [memstechtips/UnattendedWinstall#120] [General]
In  your Ventoy instructions for autounattend.xml files in the "Templates" folder, can you add several files with different names for different windows versions, i.e. 11_autounattend.xml and 10_autounattend.xml?

**Comment by memstechtips** [comment]
Hi, I haven't tested this personally.

As far as I know the file name has to stay the same so it can't be changed.
I didn't originally write those instructions and it could be an error by the initial writer.

I'll have to test this and get back.
